Tests/FunctionalTests/IdentityAdIDTests.swift

```diff
@@ -90,7 +90,7 @@ class IdentityAdIDTests: XCTestCase {
         let expectedIdentity: [String: Any] =
             [
                 "identityMap": [
-                    "ECID": [["id": propsECID, "authenticatedState": "ambiguous", "primary": 0]],
+                    "ECID": [["id": propsECID, "authenticatedState": "ambiguous", "primary": 0] as [String: Any]],
```

Comment:
Do we need to add `as [String: Any]` to just one entry? Is there a way to avoid this, since we have it declared with the variable as well?